Automattic/wp-calypso
# Update/modify renewal paths

#### Proposed Changes

* This PR modifies the cancellation flow paths for Jetpack products so that product cancellation uses the main cancellation flow instead of the auto-renew on/off toggle.
* This PR also modifies the appearance of the "Renew now" button that shows on individual purchase pages so that is shows in the chosen theme's primary color.
* This PR also adds some additional language to the purchase cancellation screen when a purchase is within the refund window.

Context: p1HpG7-j9Q-p2

#### Testing Instructions

- To test, you will need a Jetpack site with a purchased Jetpack plan/ product. Note that you should not use a purchase made with credits as the test instructions will not work properly.
- Check out this branch in your testing environment.
- Navigate to `/me/purchases` and select your Jetpack purchase from the list.
- On the purchase page, if automatic renewal is ON, the "Subscription Renewal" section of the purchase page will just show the next renewal date and no auto-renew on/off toggle.
![Screen Shot 2022-11-28 at 3 04 48 PM](https://user-images.githubusercontent.com/18016357/204370513-915a6086-6093-426a-9836-d5ba76649ea4.png)

- On the purchase page, if automatic renewal is OFF, the "Subscription Renewal" section of the purchase page will show the expiration date of the subscription and provide a link to Re-enable automatic renewal.
![Screen Shot 2022-11-28 at 12 20 35 PM](https://user-images.githubusercontent.com/18016357/204370539-b97ccdbb-3733-46c4-a9b3-ea960b34b71a.png)

- Click on the "Re-enable" link and confirm that the text that the "Subscription Renewal" section changes from showing an expiry date to showing the next renewal date and that the "Re-enable" link goes away.
- Note that on the purchase page, the "Renew Now" button shows in the chosen theme's primary color.
![Screen Shot 2022-11-28 at 3 05 47 PM](https://user-images.githubusercontent.com/18016357/204370639-90cb0de3-ff32-42ba-a07a-84e8bff61212.png)

- Click on the "Cancel Subscription" button at the bottom of the Purchase page to go to the Cancel Purchase page. 
![Screen Shot 2022-11-28 at 3 04 57 PM](https://user-images.githubusercontent.com/18016357/204370690-6334c64f-8463-4aba-9f0d-720117baf8f2.png)

- If your purchase is still within the refund window (which will be true if you made a new purchase for this test), you should see some updated copy on the cancel purchase page:
![Screen Shot 2022-11-28 at 12 33 55 PM](https://user-images.githubusercontent.com/18016357/204370745-3e97e2da-f7d2-4910-a990-ea665cbe04f0.png)

- These changes should be restricted to Jetpack purchases only. Test with a WordPress.com plan to confirm that the auto-renew on/off toggle still shows for this purchase.


#### Pre-merge Checklist
